### PR TITLE
Modified k1 value and some of the doc

### PR DESCRIPTION
**Pull request by JFLemieux73**
For detailed information about submitting Pull Requests (PRs) to the CICE-Consortium,
please refer to: <https://github.com/CICE-Consortium/About-Us/wiki/Resource-Index#information-for-developers>

## PR checklist
- [x] Short (1 sentence) summary of your PR: 
    Based on our results, k1=7.5 is a better value than 8.0. I also made very small modifications to the doc. 
- [x] Developer(s): 
    @JFLemieux73 
- [x] Suggest PR reviewers from list in the column to the right.
@apcraig 
- [x] Please copy the PR test results link or provide a summary of testing completed below.
   I did not ran the test suite...not needed here.
- How much do the PR code changes differ from the unmodified code? 
    - [ ] bit for bit
    - [ ] different at roundoff level
    - [ ] more substantial 
- Does this PR create or have dependencies on Icepack or any other models?
    - [ ] Yes
    - [x] No
- Does this PR add any new test cases?
    - [ ] Yes
    - [x] No
- Is the documentation being updated? ("Documentation" includes information on the wiki or in the .rst files from doc/source/, which are used to create the online technical docs at https://readthedocs.org/projects/cice-consortium-cice/. A test build of the technical docs will be performed as part of the PR testing.)
    - [x] Yes
    - [ ] No, does the documentation need to be updated at a later time?
        - [ ] Yes
        - [ ] No 
- [ ] Please provide any additional information or relevant details below:

### CODE REVIEW

**Review by eclare108213 — COMMENTED** [review]
This looks fine to me, but it will change the answers when this method is turned on.  Do we not have any tests in the test suite that include k1?

**Comment by apcraig** [comment]
We have short (5-10 day) tests that turn on the landfast ice but I believe no ice is grounded (do I have the terms right?).  So changing k1 could change this, but I suspect it won't.  Once the new time manager is merged and we have initial conditions for other months based on JRA55 with landfast ice on in the spinup, the idea is to create landfast tests starting in April hoping that ice will be grounded there.  See #573.

**Review by eclare108213 — APPROVED** [review]

File: doc/source/science_guide/sg_dynamics.rst [hunk 1]
```diff
@@ -240,9 +240,7 @@ CICE includes two options for calculating the seabed stress,
 i.e. the term in the momentum equation that represents the interaction
 between grounded ice keels and the seabed. The seabed stress can be
 activated by setting ``seabed_stress`` to true in the namelist. The seabed stress (or basal
-stress) parameterization of :cite:`Lemieux16` is chosen if ``seabed_stress_method``
-= ``LKD`` while the new probabilistic approach is used if ``seabed_stress_method``
-= ``probabilistic``. 
+stress) parameterization of :cite:`Lemieux16` is chosen if ``seabed_stress_method`` = ``LKD`` while the approach based on the probability of contact between the ice and he seabed is used if ``seabed_stress_method`` = ``probabilistic``.
```
Comment (phil-blain) [inline doc/source/science_guide/sg_dynamics.rst:243]: small typo here:

```suggestion
stress) parameterization of :cite:`Lemieux16` is chosen if ``seabed_stress_method`` = ``LKD`` while the approach based on the probability of contact between the ice and the seabed is used if ``seabed_stress_method`` = ``probabilistic``.
```

Comment (phil-blain) [inline doc/source/science_guide/sg_dynamics.rst:243]: small typo here:

```suggestion
stress) parameterization of :cite:`Lemieux16` is chosen if ``seabed_stress_method`` = ``LKD`` while the approach based on the probability of contact between the ice and the seabed is used if ``seabed_stress_method`` = ``probabilistic``.
```

**Pull request merged**